test/python/circuit/test_parameters.py

```diff
@@ -308,6 +309,46 @@ def test_gate_multiplicity_binding(self):
                 for gate, _, _ in qc2.data:
                     self.assertEqual(float(gate.params[0]), 1.0)
 
+    def test_calibration_assignment(self):
+        """That that calibration mapping and the schedules they map are assigned together."""
+        theta = Parameter('theta')
+        circ = QuantumCircuit(3, 3)
+        circ.append(Gate('rxt', 1, [theta]), [0])
+        circ.measure(0, 0)
+
+        rxt_q0 = pulse.Schedule(pulse.Play(
+            pulse.library.Gaussian(duration=128, sigma=16, amp=0.2*theta/3.14),
+            pulse.DriveChannel(0)))
+
+        circ.add_calibration('rxt', [0], rxt_q0, [theta])
+        circ = circ.assign_parameters({theta: 3.14})
+
+        self.assertTrue(((0,), (3.14,)) in circ.calibrations['rxt'])
+        sched = circ.calibrations['rxt'][((0,), (3.14,))]
+        self.assertEqual(sched.instructions[0][1].pulse.amp, 0.2)
+
+    def test_calibration_assignment_w_expressions(self):
```

Comment:
Can you test for partial binding and substitution with another ParameterExpression?